modules/standard/Set.chpl

```diff
@@ -441,7 +445,7 @@ module Set {
       :return: An array containing a copy of each of the elements in this set.
       :rtype: `[] eltType`
     */
-    proc const toArray(): [] eltType {
+    proc const toArray(): [] eltType where isCopyableType(eltType) {
```

Comment:
For functions like this, I generally prefer the approach of a param conditional guarding a compilation error. The error can be more friendly than the resolution error. But there might be a reason the resolution error is nicer here?

Reply:
I'm actually going to remove the where clauses from this PR and come back to them in another effort, because I just discovered the `|=` operators aren't working as expected 🤭